packages/cascara/src/ui/Table/TechnicalDesign.mdx

```diff
@@ -0,0 +1,138 @@
+---
+name: Spec
+menu: Table
+route: /table-spec
+---
+
+# Technical Design - Espressive Table
+
+# Current implementations
+
+As of `goldengirls` release, we use tables in the following applications:
+
+Barista:
+
+- Admin
+  - Service Department
+  - Service Teams
+  - Barista FAQs
+  - Users
+  - Locations
+
+Caffeine
+
+- Configuration
+- Entities
+- Localization
+- Training
+- Variables
```

Comment:
This skips mention of the "status" app.

Probably also worth noting that views like this are also going to be needed in the new version of Doppio.

There are other applications that will be needed in the future as well. Might not need to mention that, but when we are taking inventory, we probably want to include all of our application buckets. @CaffeinatedAllie will be good at helping us with this more broad view.

Reply:
ok I will add both apps to the list.

Reply:
```suggestion
- Variables
- Status
- Doppio
```